fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -610,6 +610,10 @@ private void processDropField(DropFieldClause alterClause, OlapTable olapTable,
             }
             fields.add(field);
         }
+        if (fields == null || fields.size() <= 0) {
```

Comment:
```suggestion
        if (fields.isEmpty()) {
```
`fields` must be not null.